environment.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - pdbfixer
   - pytorch-lightning
   - biopython
-  - numpy
+  - numpy==1.26.4
```

Comment:
Currently, numpy is unversioned, causing numpy 2.0+ to be installed which leads to failing tests